buildstockbatch/workflow_generator/residential_hpxml.py

```diff
@@ -31,14 +31,14 @@ def validate(cls, cfg):
         :type cfg: dict
         """
         schema_yml = """
-        measures_to_ignore: list(str(), required=False)
         build_existing_model: map(required=False)
         emissions: list(include('emission-scenario-spec'), required=False)
         utility_bills: list(include('utility-bill-scenario-spec'), required=False)
         measures: list(include('measure-spec'), required=False)
         reporting_measures: list(include('measure-spec'), required=False)
         simulation_output_report: map(required=False)
         server_directory_cleanup: map(required=False)
+        debug: bool(required=False)
```

Comment:
Introducing a "top level" debug argument that controls debug mode for: `HPXMLtoOpenStudio`, `UpgradeCosts`, and `ServerDirectoryCleanup`.